source/Userdocs/LEMSSimulation.md

```diff
@@ -7,11 +7,27 @@ For many users, the most obvious place that LEMS is used is in the LEMS Simulati
 
 See <a href="Schemas/Simulation.html">here</a> for definition of the main elements used in the file, including {ref}`schema:display`,  {ref}`schema:outputfile`, etc.
 
-
 ## Quantities and paths
 
 Specifying the quantities to save/display in a LEMS Simulation file is an important and sometimes confusing process. There is a {ref}`dedicated page <userdocs:quantitiesandrecording>` on quantities and paths in LEMS and NeuroML2.
 
+## Creating LEMS Simulation files
+
+Perhaps the easiest way to create a LEMS Simulation file is to base it off of an existing example.
+
+```{literalinclude} ./NML2_examples/lems_sim/LEMS_SimulationExample.xml
+----
+language: xml
+```
+
+Alternatively, it is possible to create a LEMS Simulation file in Python file using pyNeuroML:
```

Comment:
Should we not link to the examples we've set up in the docs themselves? They have sections on creating the simulation after the NeuroML model has been declared: https://docs.neuroml.org/Userdocs/SingleNeuronExample.html#simulating-the-model

Also, should we not put pyNeuroML first, since we want users to use that as opposed to writing LEMS xml files by hand? :thinking:

Reply:
Sure, change the order if you want. 

I just thought it was better to have short examples we can use for this purpose here. Could add links to the tutorial for more info.

Reply:
OK, sounds good. Merging this now.